lib-py/casmutils/xtal/coordinate.py

```diff
@@ -166,16 +166,26 @@ def __add__(self, other):
 
         Parameters
         ----------
-        other : Coordinate
+        other : Coordinate or MutableCoordinate
 
         Returns
         -------
-        Coordinate
+        Coordinate or MutableCoordinate
 
         """
         py_binded = self._pybind_value + other._pybind_value
         return self._from_pybind(py_binded)
 
+    def __str__(self):
+        """Returns string to print
```

Comment:
Not very helpful. `__str__` always returns a string to print. You could have an example of the format, or remove the documentation for this function.